Make command to update site domain (#404)
NationalCentreTruthReconciliation/Secure-Record-Transfer: docs/howtouse/sitename.rst

Updating the Domain of Your Site
================================

If you are running the record transfer app somewhere other than your localhost, you will need to add
your site's domain to the database. If you do not set the domain, your emails will not send
correctly, and will have the wrong link to your site.

> To set the domain of your site, run the following command from the `app` directory:

Reviewer: It should be made explicit that this needs to be run from the `app` directory inside the running container. Someone might confuse this for needing to run it locally.

.. code-block:: shell

>     python manage.py set_domain "https://my.domain.com"

Reviewer: The https:// isn't needed. See the documentation for [the Site model here](https://docs.djangoproject.com/en/4.2/ref/contrib/sites/#django.contrib.sites.models.Site).

```suggestion
    python manage.py set_domain "my.domain.com"
```

Author: Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/404/commits/ad7bc0aee0293e07106fa1437b8369f02b9921e6

> replacing `https://my.domain.com` with your site's domain.

Reviewer: I don't love how restructured text has you specify code inline, but you have to do it like this:

```suggestion
replacing code:`my.domain.com` with your site's fully qualified domain name.
```

If you also want to set an optional display name for your site, you can use the `--display-name` flag:

.. code-block:: shell

    python manage.py set_domain "https://my.domain.com" --display-name "My Site Name"

> replacing `"My Site Name"` with your desired display name.

Reviewer: This one doesn't need to be code:

```suggestion
replacing "My Site Name" with your desired display name.
```
